dialogue-core/src/main/java/com/palantir/dialogue/core/QueuedChannel.java

```diff
@@ -80,13 +83,21 @@
         this.maxQueueSize = maxQueueSize;
         this.queueSizeCounter = metrics.requestsQueued(channelName);
         this.queuedTime = metrics.requestQueuedTime(channelName);
+        this.limitedResultSupplier = () -> Futures.immediateFailedFuture(new SafeRuntimeException(
+                "Unable to make a request",
+                SafeArg.of("queueSizeEstimate", queueSizeEstimate.get()),
```

Comment:
This atomic read is arguably a bit pointless because we _should_ only hit this exception due to queueSizeEstimate > maxQueueSize, I guess it's just defensive in case we ever end up with a _different_ cause??